gstreamer/src/query.rs

```diff
@@ -208,17 +267,29 @@ impl fmt::Debug for QueryRef {
     }
 }
 
-impl ToOwned for QueryRef {
-    type Owned = GstRc<QueryRef>;
+pub unsafe trait AsMutPtr {
```

Comment:
This trait seems like a crutch, but I don't have a better suggestion so let's keep it like that and just not re-export it outside the query module :)

Maybe should be named `AsMutQueryPtr` or so, to make it clear from the name that this is about queries. OTOH if it's in the query module... so I guess just keep it all as is!

Reply:
Ok. I wasn't too happy with it neither.